### ISSUE TITLE
Node.js: [channel_args.c:338] grpc.max_send_message_length ignored: it must be >= 0

### ISSUE THREAD

**Issue by stephenplusplus** [grpc/grpc#10573]
-1 is documented as meaning unlimited, but it returns an error: https://github.com/grpc/grpc/blob/e3683cadd3a22d138e97c68af84a1680f80b8a4a/include/grpc/impl/codegen/grpc_types.h#L162

```
E0411 12:56:38.475000000 12208 channel_args.c:338] grpc.max_send_message_length ignored: it must be >= 0
E0411 12:56:38.475000000 12208 channel_args.c:338] grpc.max_receive_message_length ignored: it must be >= 0
```

*Related: https://github.com/GoogleCloudPlatform/google-cloud-node/pull/2192*

**Comment by emmenko** [comment]
Getting the same error. Is there an ETA for when the fix will be released?
Thanks 👍

**Comment by murgatroid99** [comment]
I think I can get a fix out within the next day or two.

**Comment by murgatroid99** [comment]
This should be fixed in gRPC 1.2.4

**Comment by stephenplusplus** [comment]
Looks like it works, thanks for the quick fix @murgatroid99!
